toc/rfc/rfc-draft-stemcell-release-schedule.md

```diff
@@ -0,0 +1,26 @@
+# Meta
+[meta]: #meta
+- Name: Stemcell Release Schedule
+- Start Date: 2022-06-29
+- Author(s): @thomasthal
+- Status: Draft <!-- Acceptable values: Draft, Approved, On Hold, Superseded -->
+- RFC Pull Request: https://github.com/cloudfoundry/community/pull/331
+
+
+## Summary
+This RFC proposes a well defined release schedule for the CF Stemcell to ensure the availability of OS related security patches in a timely manner.
+
+## Problem
+Almost daily, Canonical releases security patches of known vulnerabilities. As a PaaS offering provider, you want to consume those patches as fast as possible to mitigate vulnerabilities and keep your customers safe. Cloud Foundry has the huge security advantage to be able to rollout zero downtime updates and patch security vulnerabilities at any time. This potential is limited by the fact, that often there is only an "outdated" stemcell available, which does not yet contain latest patches.
+
+A second concern is that the current stemcell release strategy seems to be an undocumented, informal agreement. This makes it impossible to evaluate if the process to create a new stemcell is executed accordingly. Moreover, it is difficult to provide clear statements to customers how long the rollout of an OS patch takes or to judge if compliance requirements are met.
+
+## Proposal
+- Release a new stemcell every week on Tuesday, that contains all available ubuntu security patches 
+- Release a new stemcell for every critical security vulnerability as soon as possible

```

Comment:
The current stemcell pipeline is built to look at the severity level provided by Canonical and creates a notification for every USN with a __high__ priority. For these, a stemcell is already cut as soon as possible.